types/rdfjs__term-set/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for @rdfjs/term-set 1.0
+// Project: https://github.com/rdfjs-base/term-set
+// Definitions by: tpluscode <https://github.com/tpluscode>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Term } from "rdf-js";
+
+declare namespace TermSet {
+    interface TermSet<T extends Term = Term> extends Set<T> {
```

Comment:
Looks like `Set.keys()` is the only function not implemented. (Which is a bit weird for a set, but is there...)

Reply:
Set does not have a `keys` method: https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Set

Reply:
> **`Set.prototype.keys()`**
> Is the same function as the `values()` function and returns a new `Iterator` object that contains the values for each element in the `Set` object in insertion order.

Reply:
Oh, I only looked at the listing on the right. Weird

Reply:
rdfjs-base/term-set#1